.circleci/config.yml

```diff
@@ -110,7 +110,6 @@ setup_project: &setup_project
       working_directory: ~/project
       command: |
         pip install --progress-bar off -e .
-        pip install -r requirements/dev.txt
```

Comment:
Does this mean the mypy related test won't execute on CI?

Reply:
Yes. I was typing an answer as to why I think this was OK, but I actually changed my mind in the middle of writing it lol. I'll add this back.